packages/trader/src/Modules/Contract/Containers/__tests__/contract.spec.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import Contract from '../contract';
+import TraderProviders from '../../../../trader-providers';
+import { createMemoryHistory } from 'history';
+
+jest.mock('react-router', () => ({
+    ...jest.requireActual('react-router'),
+    withRouter: (component: React.FC) => component,
```

Comment:
It fails, that's why I added it

Reply:
As for more detailed problem: There's `match` inside the component which couldn't be mocked or passed correctly anyhow as it gets overwritten during test execution by router itself. The only way to fix it was to mock `withRouter` itself, otherwise the component couldn't be tested at all. I think the following issue with `BrowserRouter` is also because of the incorrect `match` passing.